TeachingRecordSystem/tests/TeachingRecordSystem.TestCommon/TestData.CreatePerson.cs

```diff
@@ -256,8 +257,21 @@ public CreatePersonBuilder WithTrn()
             return this;
         }
 
-        public CreatePersonBuilder WithGender(Contact_GenderCode? gender)
+        public CreatePersonBuilder WithGender(bool hasGender = true)
         {
+            _hasGender = hasGender;
+
+            if (_hasGender is false)
+            {
+                _gender = null;
+            }
+
+            return this;
+        }
+
+        public CreatePersonBuilder WithGender(Contact_GenderCode gender)
```

Comment:
Better to use our model types instead of CRM ones
```suggestion
        public CreatePersonBuilder WithGender(Gender gender)
```

Reply:
OK fair, I was thinking it made sense as we're creating a CRM person first and syncing it but better to use the types the data will have after migration

Reply:
Ah @gunndabad this method already existed using the CRM enum, the diff got confused. I just changed it slightly. Do you think it should be changed to use the model enum? It's used a couple of times in `Api.IntegrationTests` and `CrmIntegrationTests`